web/modules/custom/helloworld/helloworld.info.yml

```diff
@@ -0,0 +1,9 @@
+name: Hello world
+type: module
+description: My custom Hello World module for Drupal 10.
+package: Custom Module
+version: 1.0
+core_version_requirement: ^10
+dependencies:
+  - drupal:views
+  - drupal:media
```

Comment:
This red circle here means new line is missing at the end of file.
So please make sure to include the newline.